Project/frontend/src/components/Graph/index.css

```diff
@@ -1,4 +1,11 @@
-.graph_container {
+.container {
```

Comment:
Please use a namespace for this class something like graph_main_container. As this name is too generic it might override the styles if the same class is used somewhere else. It is better to namespace CSS classes.

Reply:
It's not used in the project, I think I changed the name but forgot to delete it from there, so I deleted it now.